extract_endpoint/tests/test_endpoint.py

```diff
@@ -0,0 +1,165 @@
+import io
+import base64
+import pytest
+from flask import testing
+from azure.storage import blob
+from extract_endpoint import azure_utils
+from extract_endpoint import crypt_utils
+from extract_endpoint import endpoint
+
+
+endpoint.App.testing = True
+
+
+@pytest.fixture
+def azure_service() -> blob.BlockBlobService:
+    return blob.BlockBlobService(account_name=azure_utils.AZURE_EMULATOR_COORDS.account,
```

Comment:
It didn't occur to me until now, but does it make more sense to have the `AZURE_EMULATOR_COORDS` defined in one of the test files, instead of in a src file? Unless I am mistaken, it's only ever used in tests

Reply:
Perhaps, though they're needed in all of the test files in this project, so we'd be importing one test file into the other test files. Seems fugly.

Reply:
It doesn't have to be stored in one of the test files that actually contains tests, it can be in just another .py file in the tests directory that the other tests can import

Reply:
Also a good candidate for a test fixture in `conftest.py`

Reply:
👍